### ISSUE TITLE
Add icon to quote a message

### ISSUE THREAD

**Issue by contrapunctus-1** [conversejs/converse.js#1195]
~Current behavior -~
~1. Type something in the entry box without sending it~
~2. Press Up enough times to switch edit/reuse a previous message (you aren't likely to do this after Step 1, but it often happens by accident, especially in multi-line messages)~
~OR~
~Press Down 1-2 times when you're on the last line (more likely to happen)~
~3. Press Down to get back to the message you were typing - the message you were typing has vanished and you have a blank box again.~

~That's really frustrating, especially if you were typing a big message :)~

~Suggestion - if you press Up, restore the draft when you get back to it via pressing Down. Don't wipe the message box when you press Down with the cursor at the end of the last line.~

Complementary improvement - just as hovering over messages displays a pencil button to edit, hovering * should also display a quote button to quote that message. (* And perhaps also when selecting some text.)

**Comment by Nyco** [comment]
Isn't this one related? https://github.com/conversejs/converse.js/issues/1142

**Comment by jcbrand** [comment]
Yes, this is a duplicate of #1142, although we can keep it open for the "quote message" feature you suggsted.
